pages/new-hackers.md

```diff
@@ -8,3 +8,109 @@ permalink: /new-hackers/
+                </li>
+                <li>
+                    <b>Pull</b> brings changes in a different repository than the one currently being used. This merges those changes into the repository.
+                </li>
+                <li>
+                    <b>Pull Request</b> is when you have made changes to a repository that is owned by someone else in your own local version and you are <em>requesting</em> that they incorporate those changes into their repository.
+                </li>
+                <li>
+                    <b>Push</b> takes your local changes and gives them to a remote repository most commonly the repository hosted on Github.
+                </li>
+                <li>
+                    <b>Fork</b> creates a new version of a repository that allows you to develop while having the freedom of not being tied down to the original version of the repository.
+                </li>
+            </ul>
+        </p>
+        <iframe width="640" height="360" src="https://www.youtube.com/embed/_ALeswWzpBo" frameborder="0" gesture="media" allowfullscreen></iframe>
+    </li>
+    <li> 
+        <h1> Let's create a Github account! </h1>
+        <p>
+            Talk is cheap. Let's create our own Github account so we can start developing our own project: a personal website!
+            <br>
+            <a href='https://github.com' target="_blank"> Click here to sign up.</a>
+        </p>
+    </li>
+    <li>
+        <h1> Let's fork this repository </h1>
+        <p>
+            <a href='https://github.com/21mooie/small-web-projects' target="_blank"> Go to the repository here. </a>
+            <br>
+            Click on the fork button in the top right corner.
+            <img src="{{ "images/Fork_Screenshot.png" | relative_url }}"  alt="Screenshot of Github Fork button" class="img-responsive"/>
+        </p>
+    </li>
+    <li>
+        <h1> The Github Desktop Client</h1>
+        <p>
+            Normally, developers will use the command line during development to invoke specific commands such as Git's commands. It can be hard to remember these when you are just starting out with development so often it is nice to have a graphical tool that has the power to execute the same commands on the command line. The desktop client is just that tool. It will give us the power to commit changes, create new branches, push to our Github repository, pull changes from our Github repository all without touching a terminal. 
+            <br>
+            <a href='https://desktop.github.com/' target="_blank">Let's download the Github desktop client</a>
+            <br>
+            Login to your Github account with the desktop client then pull your repository down to your local machine.
+        </p>
+    </li>
+    <li>
+        <h1> A Tale of three text editors </h1>
```

Comment:
Similar story with the text editors. Perhaps we could have a separate resource page for people who are just starting to code. I like the three editors by the way!